lib/entities/request/index.ts

```diff
@@ -27,6 +27,7 @@ export interface QuoteRequestInfo {
   type: TradeType;
   slippageTolerance?: string;
   swapper?: string;
+  forceUniswapX?: boolean;
```

Comment:
prefer `use/enableUniswapX` as a name as this doesn't force it given it still passes through many filter stages so not actually forced